solutions/wilson_romero/solution.py

```diff
@@ -0,0 +1,18 @@
+""" Module solution """
+from typing import List
+
+class Solution:
+    """ Challenge solution """
+    def duplicate_zeros(self, arr: List[int]):
+        """Duplicate zeros and shift right"""
+        next_index = 0
+        length = len(arr)
+        while True:
+            try:
+                next_index = arr.index(0, next_index)
+                arr.insert(next_index + 1, 0)
```

Comment:
Watch out with this operation. Your solution isn't optimum, its O(n^2) in time and O(n) in space. There's a O(n) in time and O(1) in space solution.